slidesimport/slidesimport.py

```diff
@@ -88,7 +88,9 @@ def run(rawArgs=None):
     print "Reading files ..."
     try:
         notesFilePath = os.path.expandvars(os.path.expanduser(args.notes))
-        notesQuestions = Parser(file(notesFilePath, 'r')).getQuestions()
+        # notesQuestions = Parser(file(notesFilePath, 'r')).getQuestions()
+        (notesQuestions, notesQWS, notesQFBS, notesSFBQ, notesAWS, notesAFBS, notesSFBA) = \
+            Parser(file(notesFilePath, 'r')).getQAndAParsing()
```

Comment:
Perhaps a [namedtuple](https://docs.python.org/2/library/collections.html#collections.namedtuple) would be a better fit?